### PR TITLE
chapter 6

### PR DESCRIPTION
**Pull request by starocketeer**

### CODE REVIEW

**Comment by starocketeer** [comment]
Ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/users.scss]: remove empty file

File: app/helpers/users_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (ngocvt-0484) [inline app/helpers/users_helper.rb]: remove empty file

File: config/routes.rb [hunk 3]
```diff
@@ -1,10 +1,10 @@
 Rails.application.routes.draw do
   root "static_pages#home"
   
-  get "/home", to: "static_pages#home"       
-  get "/help", to: "static_pages#help"
-  get "/about", to: "static_pages#about"
-  get "/contact", to: "static_pages#contact"
-  get "/signup", to: "user#new"
+  get   "/home", to: "static_pages#home"
```
Comment (ngocvt-0484) [inline config/routes.rb]:
```suggestion
  get "/home", to: "static_pages#home"       
```

File: config/settings.yml [hunk 4]
```diff
@@ -10,3 +10,13 @@ links:
...
+
+validations:
+  name:
+    max_length: 50
+  email:
+    max_length: 255
+    regex: !ruby/regexp '/\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i'
+  password:
+    min_length: 6
+
```
Comment (ngocvt-0484) [inline config/settings.yml]: remove this line

**Comment by ngocvt-0484** [comment]
please change 6 commits to 1 commit

File: app/assets/stylesheets/users.scss (hunk 1 above)
Comment (starocketeer) [reply app/assets/stylesheets/users.scss]: Done

File: app/helpers/users_helper.rb (hunk 2 above)
Comment (starocketeer) [reply app/helpers/users_helper.rb]: Done

File: config/routes.rb (hunk 3 above)
Comment (starocketeer) [reply config/routes.rb]: Done

**Comment by starocketeer** [comment]
Ready

File: config/settings.yml (hunk 4 above)
Comment (starocketeer) [reply config/settings.yml]: Done

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**